### PR TITLE
[IDB IGNORE] [MDB IGNORE] Euthenia final stretch

### PR DESCRIPTION
**Pull request by ktoma36**
## About The Pull Request

(Hopefully) The final Stretch before I can start doing proper playtest and eventually the final merge of Euthenia

### CODE REVIEW

**Comment by ktoma36** [comment]
My brain is goo, apologies for slow progress

File: code/modules/recycling/conveyor2.dm [hunk 1]
```diff
@@ -29,6 +29,9 @@
 /obj/machinery/conveyor/centcom_auto
 	id = "round_end_belt"
 
+
+/obj/machinery/conveyor/on
```
Comment (TheLordME) [inline code/modules/recycling/conveyor2.dm:33]: ????

File: code/modules/recycling/conveyor2.dm [hunk 2]
```diff
@@ -218,6 +221,8 @@
 	var/list/conveyors		// the list of converyors that are controlled by this switch
 	anchored = 1
 
+/obj/machinery/conveyor_switch/two_way_on
+	position = 1
```
Comment (TheLordME) [inline code/modules/recycling/conveyor2.dm:225]: I am not sure why we would want default on conveyors/conveyor switches

File: code/modules/recycling/disposal.dm [hunk 3]
```diff
@@ -493,6 +493,38 @@
 	else
 		return ..(mover, target)
 
+
+
+/obj/machinery/disposal/wall
+	name = "inset disposal unit"
```
Comment (TheLordME) [inline code/modules/recycling/disposal.dm:499]: did you test if this works? I am not sure if the bump proc is called when you are on a wall tile

**Review by TheLordME — COMMENTED** [review]
cant really look at the map with MDB ingnore on

File: code/modules/recycling/disposal.dm (hunk 3 above)
Comment (ktoma36) [reply code/modules/recycling/disposal.dm:499]: They work quite well, ive not noticed anything with bumping into them though. Is the bump proc tripping into one?

File: code/modules/recycling/conveyor2.dm (hunk 2 above)
Comment (ktoma36) [reply code/modules/recycling/conveyor2.dm:225]: Mostly so I didnt have to make a var edit version, I wanted one that would be default on

File: code/modules/recycling/conveyor2.dm (hunk 1 above)
Comment (ktoma36) [reply code/modules/recycling/conveyor2.dm:33]: Ah forgot about that, removing it now

**Review by TheLordME — APPROVED** [review]

**Pull request merged**